api/inventory/v1alpha1/inventory_types.go

```diff
@@ -138,6 +138,11 @@ type InventorySpec struct {
 	// KubeRbacProxyImage is the full reference of the container image that implements the kube-rbac-proxy.
 	//+optional
 	KubeRbacProxyImage *string `json:"kubeRbacProxyImage"`
+
+	// PostgresImage is the full reference of the container image that implements the Postgres database.
+	//+optional
+	PostgresImage *string `json:"postgresImage"`
```

Comment:
Generally images are set using the operator env vars. Having it in env also signals users that they may not swap it out/not-configurable. Maybe this can be removed?

Reply:
Ok, I'm in favor of removing it given that we have the env variable method available to us.   I'll also take out the kube-rbac-proxy one while i'm at it.

Reply:
changed